### PR TITLE
 KFRD-16 Style checker was added 

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
[run tests]

**Comment by placccebo** [comment]
[run tests]

File: project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version = 0.13.16
```
Comment (placccebo) [inline project/build.properties]: update to the latest version

**Review by placccebo — CHANGES_REQUESTED** [review]

File: project/build.properties (hunk 1 above)
Comment (MedvedevBW) [reply project/build.properties]: ok

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**